### PR TITLE
Template merge

### PR DESCRIPTION
**Pull request by sven1103**
This PR introduces the initial project skeleton based on QUBE, which holds the current templates for QBiC software projects.

Future PRs will be smaller!

This allows us to enable syncing, once a new template version was released with QUBE. Thus, we can track changes in the template and incorporate them into QBiC software products, using the Github workflow `sync_project`, that is provided with this PR (`.github/sync_project.yml`).

### CODE REVIEW

**Comment by Zethson** [comment]
@sven1103 since the sync workflow is failing with  a credentials error - how did you setup the secret?

**Comment by Zethson** [comment]
+ Note that the TEMPLATE branch should have never seen the sync workflow.

We may run into self modifying workflow issues elsewise.

**Comment by Zethson** [comment]
@sven1103 Yeah I thought about using Jonny for sync tasks. Since QBiC is also "centralized" for any qbicsoftware repositories this should work.

-> If we use Jonny no QBiC member can create repositories outside of qbicsoftware with sync support (e.g. PhD students)
-> If this is fine we can hardcore jonny + his secret.

Thoughts?

File: .github/workflows/sync_project.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          with:
+              fetch-depth: 0
+          name: Check out source-code repository
+
+        - uses: oleksiyrudenko/gha-git-credentials@v1
+          with:
+               name: 'sven1103'
+               email: 'sven.fillinger@qbic.uni-tuebingen.de'
+               actor: 'sven1103'
+               token: '${{ secrets.QUBE_SYNC_TOKEN}}'
```
Comment (jenniferboedker) [inline .github/workflows/sync_project.yml]: Is this the standard then? Or does this need to be updated by everyone pulling the code?

Comment (Zethson) [reply .github/workflows/sync_project.yml]: @jenniferboedker We will hardcode Johnny and you do no need to adapt anything.

**Review by jenniferboedker — APPROVED** [review]
LGTM, but solely checked who concise files describing how PRs are written. 
Since my experience in Github Actions is low I would leave that to @Zethson.

File: .github/workflows/sync_project.yml (hunk 1 above)
Comment (jenniferboedker) [reply .github/workflows/sync_project.yml]: Ah ok, so Johnny is the developer of QBiC in Github?

Comment (Zethson) [reply .github/workflows/sync_project.yml]: Johnny is the best and most hardworking developer of QBiC. 
He never complains and just does what he is supposed to do (most of the times).

**Pull request merged**